### DISCUSSION TITLE
我就想看看 官方怎么解决？

### DISCUSSION THREAD

**Discussion by jean-reno** [getlantern/lantern-forum-cn#833] [General]
链接不上去 免费的VPN都比你们快 你们收费这么多年 对得起老用户吗？这个钱 你们花的安心吗 》？

**Comment by MaxMKMax** [comment]
@jean-reno 您好. 请您提供我们log. 我们看是出现什么问题. 暂时没有别的用户遇到像您的问题. 我们需要log提交给技术部门. 帮我们提交各应用内报告吧. 然后给我 6 位数票号. 谢谢!
